src/components/Dashboard.vue

```diff
@@ -177,9 +177,20 @@
           data-cy="expenses"
         >
         </AdjustCounter>
-        <InfoButton
-          link="https://www.cgd.pt/Site/Saldo-Positivo/leis-e-impostos/Pages/deducoes-especificas.aspx#:~:text=Empresariais%20e%20Profissionais%20(Categoria%20B)&text=Se%20estiver%20enquadrado%20no%20regime,bruto%20(antes%20dos%20descontos)."
-        />
+        <span>
```

Comment:
love the transition! can you make the "auto" button and the info button horizontal aligned, with some space between them? I think it would look better. Something like this:

```html

<span class="flex gap-2">
<InfoButton
...
```

or whatever you think it's best!